src/template.rs

```diff
@@ -25,8 +27,24 @@ impl Template {
 No changes. Kubernetes configurations are up-to-date.
 ```
 {{else}}
-* updated
-{{ changed_kinds }}
+{{#if (gt (len created_kinds) 0)}}
+* created
+{{#each created_kinds}}
+  * {{this}}
+{{/each}}
+{{/if}}
+{{#if (gt (len pruned_kinds) 0)}}
+* pruned
+{{#each pruned_kinds}}
+  * {{this}}
+{{/each}}
+{{/if}}
+{{#if (gt (len configured_kinds) 0)}}
+* configured
+{{#each configured_kinds}}
+  * {{this}}
+{{/each}}
```

Comment:
[comment] @kitagry 
Did you consider using H2 (or smaller one) as a separator?
If not, I think it is good to compare the format.

```suggestion
{{#if (gt (len created_kinds) 0)}}
## created
{{#each created_kinds}}
  * {{this}}
{{/each}}
{{/if}}
{{#if (gt (len pruned_kinds) 0)}}
## pruned
{{#each pruned_kinds}}
  * {{this}}
{{/each}}
{{/if}}
{{#if (gt (len configured_kinds) 0)}}
## configured
{{#each configured_kinds}}
  * {{this}}
{{/each}}
```